src/dialogs/ConfigDialog.cpp

```diff
@@ -95,13 +95,15 @@ class GeneralPanel : public QWidget
 
     mPushCommit = new QCheckBox(tr("Push after each commit"), this);
     mPullUpdate = new QCheckBox(tr("Update submodules after pull"), this);
+    mAutoPrune = new QCheckBox(tr("Prune when fetching"), this);
 
     QFormLayout *form = new QFormLayout(this);
     form->addRow(tr("User name:"), mName);
     form->addRow(tr("User email:"), mEmail);
     form->addRow(tr("Automatic actions:"), fetchLayout);
     form->addRow(QString(), mPushCommit);
     form->addRow(QString(), mPullUpdate);
+    form->addRow(QString(), mAutoPrune);
```

Comment:
I am wondering if it would not be more intuitive to have this new checkbox close to the "Fetch every" checkbox, even though it applies on more cases than just auto fetch. It would sort of group all fetch related settings.